<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>828</width>
    <height>478</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>天气预报</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QWidget" name="widget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>810</width>
       <height>460</height>
      </size>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget#widget{
	border-image: url(:/images/weaUI.png);
	color:rgb(255, 255, 255);
}

QLabel{
	font: 25 10pt &quot;Microsoft YaHei&quot;;
	border-radius: 4px;
	background-color: argb(60, 60, 60, 130);
	color: rgb(255, 255, 255);
}

Line{
	background-color: rgb(0, 85, 0);
}



</string>
     </property>
     <widget class="QLineEdit" name="cityLineEdit">
      <property name="geometry">
       <rect>
        <x>575</x>
        <y>10</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">foreground-color: rgb(255, 0, 0);
background-color: rgb(0,0,0);
</string>
      </property>
      <property name="placeholderText">
       <string>城市</string>
      </property>
     </widget>
     <widget class="Line" name="hLine_1">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>45</y>
        <width>781</width>
        <height>1</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="Line" name="hLine_10">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>183</y>
        <width>351</width>
        <height>1</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="Line" name="hLine_11">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>335</y>
        <width>351</width>
        <height>1</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="Line" name="vLine_1">
      <property name="geometry">
       <rect>
        <x>205</x>
        <y>340</y>
        <width>1</width>
        <height>101</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QLabel" name="dateLb">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>3</y>
        <width>421</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 20pt &quot;Adobe 黑体 Std R&quot;;
color: rgb(255, 255, 255);
background-color: rgb(255, 255, 255, 0);</string>
      </property>
      <property name="text">
       <string>2023-01-01</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>10</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 20pt &quot;华文琥珀&quot;;
background-color: rgba(255, 255, 255, 0);
color: rgb(85, 255, 0);</string>
      </property>
      <property name="text">
       <string>天气预报</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="leafLb">
      <property name="geometry">
       <rect>
        <x>18</x>
        <y>58</y>
        <width>20</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">image: url(:/icons/leaf.ico);
background-color: argb(255, 255, 255, 0);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="noticeLb">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>160</y>
        <width>351</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
background-color: argb(60, 60, 60, 0);</string>
      </property>
      <property name="text">
       <string>阳光总在风雨后</string>
      </property>
     </widget>
     <widget class="QLabel" name="qualityLb">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>58</y>
        <width>56</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: argb(255, 255, 255, 0);</string>
      </property>
      <property name="text">
       <string>优</string>
      </property>
     </widget>
     <widget class="QPushButton" name="refreshBt">
      <property name="geometry">
       <rect>
        <x>753</x>
        <y>10</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">image: url(:/icons/refresh.ico);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="res.qrc">
        <normaloff>:/icons/refresh.ico</normaloff>:/icons/refresh.ico</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>31</width>
        <height>31</height>
       </size>
      </property>
     </widget>
     <widget class="QToolButton" name="searchBt">
      <property name="geometry">
       <rect>
        <x>722</x>
        <y>15</y>
        <width>20</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/icons/search.ico);</string>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
     <widget class="QLabel" name="sunRiseSetLb">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>340</y>
        <width>150</width>
        <height>100</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="temLb">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>241</width>
        <height>101</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
font: 50pt &quot;Arial&quot;;</string>
      </property>
      <property name="text">
       <string>26°</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="typeIcoLb">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>63</y>
        <width>101</width>
        <height>101</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/day/晴.png);
background-color: argb(60, 60, 60, 0);

</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="typeLb">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>120</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
background-color: argb(60, 60, 60, 0);</string>
      </property>
      <property name="text">
       <string>晴</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QWidget" name="gridLayoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>190</y>
        <width>351</width>
        <height>141</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="5" column="1">
        <widget class="QLabel" name="flLb">
         <property name="text">
          <string>&lt;3级</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="flTitleLb">
         <property name="text">
          <string>风力</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="fxTitleLb">
         <property name="text">
          <string>风向</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="shiduLb">
         <property name="text">
          <string>86%</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLabel" name="fxLb">
         <property name="text">
          <string>东南风</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="cityLb">
         <property name="text">
          <string>某某区</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="shiduTitleLb">
         <property name="text">
          <string>湿度</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="cityTitleLb">
         <property name="text">
          <string>城市</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="gridLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>50</y>
        <width>421</width>
        <height>381</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="2" column="3">
        <widget class="QLabel" name="date3Lb">
         <property name="text">
          <string>01/04</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QLabel" name="week3Lb">
         <property name="text">
          <string>后天</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QLabel" name="date2Lb">
         <property name="text">
          <string>01/03</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLabel" name="week2Lb">
         <property name="text">
          <string>明天</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="week1Lb">
         <property name="text">
          <string>今天</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="4">
        <widget class="QLabel" name="date4Lb">
         <property name="text">
          <string>01/05</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="date1Lb">
         <property name="text">
          <string>01/02</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLabel" name="quality1Lb">
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 0);</string>
         </property>
         <property name="text">
          <string>4级</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="4">
        <widget class="QLabel" name="week4Lb">
         <property name="text">
          <string>大后天</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QLabel" name="type1Lb">
         <property name="text">
          <string>中雨</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QLabel" name="typeIco1Lb">
         <property name="styleSheet">
          <string notr="true">image: url(:/day/大暴雨到特大暴雨.png);</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QLabel" name="high1Lb">
         <property name="text">
          <string>9°</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="QLabel" name="quality2Lb">
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 170, 0);</string>
         </property>
         <property name="text">
          <string>4级</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="5" column="2">
        <widget class="QLabel" name="type2Lb">
         <property name="text">
          <string>晴</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="6" column="2">
        <widget class="QLabel" name="typeIco2Lb">
         <property name="styleSheet">
          <string notr="true">image: url(:/day/晴.png);</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="7" column="2">
        <widget class="QLabel" name="high2Lb">
         <property name="text">
          <string>9°</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="4" column="3">
        <widget class="QLabel" name="quality3Lb">
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 0, 0);</string>
         </property>
         <property name="text">
          <string>4级</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="5" column="3">
        <widget class="QLabel" name="type3Lb">
         <property name="text">
          <string>多云</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="6" column="3">
        <widget class="QLabel" name="typeIco3Lb">
         <property name="styleSheet">
          <string notr="true">image: url(:/night/雾.png);</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="7" column="3">
        <widget class="QLabel" name="high3Lb">
         <property name="text">
          <string>9°</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="4" column="4">
        <widget class="QLabel" name="quality4Lb">
         <property name="styleSheet">
          <string notr="true">color: rgb(170, 0, 0);</string>
         </property>
         <property name="text">
          <string>4级</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="5" column="4">
        <widget class="QLabel" name="type4Lb">
         <property name="text">
          <string>阴</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="6" column="4">
        <widget class="QLabel" name="typeIco4Lb">
         <property name="styleSheet">
          <string notr="true">image: url(:/day/阵雨.png);</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="7" column="4">
        <widget class="QLabel" name="high4Lb">
         <property name="text">
          <string>9°</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="10" column="1">
        <widget class="QLabel" name="low1Lb">
         <property name="text">
          <string>6°</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="10" column="2">
        <widget class="QLabel" name="low2Lb">
         <property name="text">
          <string>6°</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="10" column="3">
        <widget class="QLabel" name="low3Lb">
         <property name="text">
          <string>6°</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="10" column="4">
        <widget class="QLabel" name="low4Lb">
         <property name="text">
          <string>6°</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="8" column="1" rowspan="2" colspan="4">
        <widget class="QLabel" name="curveLb">
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="3" column="1" colspan="4">
        <widget class="Line" name="line_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="5">
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="7" rowspan="12">
        <widget class="Line" name="line_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item row="11" column="1" colspan="4">
        <widget class="Line" name="line_5">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="1" column="0" rowspan="11">
        <widget class="Line" name="line_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="gridLayoutWidget_4">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>50</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="0" column="1">
        <widget class="QLabel" name="provinceLb">
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);
font: 16pt &quot;Microsoft YaHei UI&quot;;
background-color: rgba(60, 60, 60, 0);</string>
         </property>
         <property name="text">
          <string>东京</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="styleSheet">
          <string notr="true">background-color: argb(60, 60, 60, 0);
image: url(:/icons/location.ico);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
